### PR TITLE
Postion stack test cases

### PR DESCRIPTION
**Pull request by abhishek9celestial**
Postionstack test cases for the following pages:
1. Home page
2. Pricing Page
3. Affiliate Page
4. Documentation Page
5. FAQ Page
6. Login Page
7. Sign up Page
8. Status Page

### CODE REVIEW

File: cypress/e2e/Positionstack/Affiliate.cy.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    it('1. Test to click on the "Affiliate" link from the header', () => {
+        cy.get('.header > .container > ul > :nth-child(4) > a').click();
+    });
+
+    it('2. Test to check for the "Earn Passive Income with" text is present', () => {
+        cy.componentVisiblityCheck('h1', 'Earn Passive Income with');
+    });
+
+    it('3. Test to check for the earn commisions text caption is present', () => {
```
Comment (Karthikkdl) [inline cypress/e2e/Positionstack/Affiliate.cy.js]: Can you tell me the issue here? @abhishek9celestial

File: cypress/e2e/Positionstack/FAQ.cy.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    it('2. Test to show the "FAQs" page and check the text "Frequently Asked Questions"', () => {
+        cy.componentVisiblityCheck('h2','Frequently Asked Questions');
+    });
+
+    it('3. Test to check "Genral Question" section', () => {
+        cy.componentVisiblityCheck('.container > :nth-child(1) > h3', 'General');
+    });
+
+    it('4. Test to check the "What is postionstack?" Question ', () => {
```
Comment (Karthikkdl) [inline cypress/e2e/Positionstack/FAQ.cy.js]: Question?? and why there's space after that?

File: cypress/e2e/Positionstack/FAQ.cy.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        cy.componentVisiblityCheck(':nth-child(1) > ul > :nth-child(1) > .question', 'What is positionstack?');
+        cy.get(':nth-child(1) > ul > :nth-child(1) > .question').click();
+    });
+
+    it('5. Test to check "Geocoding & API" Question section', () => {
```
Comment (Karthikkdl) [inline cypress/e2e/Positionstack/FAQ.cy.js]: Question?

File: cypress/e2e/Positionstack/Status.cy.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
+describe('Positionstack status page', () => {
+    before(() => {
+        cy.navigateUrlwithCookies('https://status.positionstack.com/');
+        cy.wait(2000);
```
Comment (NaveenK15) [inline cypress/e2e/Positionstack/Status.cy.js]: remove this

File: cypress/e2e/Positionstack/Status.cy.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+        cy.componentVisiblityCheck('.psp-monitor-row');
+    });
+
+    it('9. Test to check "Calendar view" button present', () => {
+        cy.componentVisiblityCheck('.psp-calendar-link', 'Calendar view')
+    });
+
+    it('10. Test to click on "Calendar view" button', () => {
+        cy.get('.psp-calendar-link').click();
+        // cy.wait(2000);
```
Comment (NaveenK15) [inline cypress/e2e/Positionstack/Status.cy.js]: need code clean up

File: cypress/e2e/Positionstack/SignUp.cy.js [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+        cy.get('#tax_id').type('123-45-6789');
+        cy.get('#HowDidYouHear');
+        cy.get('iframe').then(($iframe) => {
+            const $body = $iframe.contents().find('body');
+            cy.wrap($body).find('.recaptcha-checkbox').click();
+        });
+        cy.get('.submit').should('not.be.disabled');
+    });
+
+    xit('18. Test to check the alert message "Please enter a password of at least 6 and maximum 48 characters"', () => {
```
Comment (NaveenK15) [inline cypress/e2e/Positionstack/SignUp.cy.js:116]: xit? why

**Review by NaveenK15 — APPROVED** [review]

**Pull request merged**